src/website/app/Label.js

```diff
@@ -20,28 +20,29 @@ import { createStyledComponent } from '../../utils';
 
 type Props = {
   children: React$Node,
-  variant?: string
+  variant?: 'regular' | 'success' | 'warning' | 'danger'
 };
 
 const Root = createStyledComponent('span', ({ theme, variant }) => {
   const backgroundColor =
     variant === 'regular'
-      ? theme.color_gray_50
+      ? theme.color_theme_60
       : theme[`backgroundColor_${variant}`];
 
   const color =
-    variant === 'regular' ? theme.color_text : theme[`color_text_on${variant}`];
+    variant === 'regular'
+      ? theme.color_text_onprimary
+      : theme[`color_text_on${variant}`];
 
   return {
     backgroundColor,
     borderRadius: theme.borderRadius_1,
+    bottom: '0.15em', // adjusted visually for middle vertical alignment
```

Comment:
Super minor, but I’ve been using “optical adjustment” in these sort of comments. We should be consistent for easy finding later.